util/event_parser.l

```diff
@@ -0,0 +1,85 @@
+%option noyywrap
```

Comment:
I would not mind a small review of this together. I don't do Lex and Yacc on a daily basis 😅

Reply:
@sanchda could we have a small doc to describe the syntax ?